react/react-addons-test-utils.d.ts

```diff
@@ -6,7 +6,7 @@
 /// <reference path="react.d.ts" />
 
 declare namespace __React {
-    interface SyntheticEventData {
+    interface SyntheticEventData extends Event {
```

Comment:
Would you copy all of the properties of `Event` onto `SyntheticEventData` and have them as optional?